Cargo.toml

```diff
@@ -91,9 +91,9 @@ name = "fish_key_reader"
 path = "src/bin/fish_key_reader.rs"
 
 [features]
-default = ["installable"]
+default = ["embed-data"]
```

Comment:
My `cargo install --path . --debug && ~/.cargo/bin/fish` tries to source files from `/usr/share/fish/functions`
This is because I have this hack in my config:

	set -a fish_complete_path /usr/share/fish/vendor_completions.d /usr/share/fish/completions
	set -a fish_function_path /usr/share/fish/vendor_functions.d/ /usr/share/fish/functions

of course the added "/usr/share/fish/functions" is wrong, I'm not sure why I added it

I think the vendor_completions.d used to be necessary for running installations to ~/.local,
but they should not be necessary if I set `XDG_DATA_DIRS`.. probably there's a reason I still have it

Reply:
The installable builds never read the vendor stuff, yes.

I'm not sure if it is a good idea to do that or not, because these builds could be much newer than what the vendor stuff is prepared for.

For now, I'm explicitly punting on that for later - if we end up making this style of build the default, we'll have to handle it, but until then it's the status quo.